boards/sltb001a/Makefile.dep

```diff
@@ -0,0 +1,8 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+    USEMODULE += saul_gpio
+    USEMODULE += bmp280
+    USEMODULE += si7021
+endif
+
+# i2c is required for environmental sensors
+USEMODULE += periph_i2c
```

Comment:
The sensors should specify their dependencies themselves, in drivers/Makefile.dep.

Reply:
The board itself needs i2c for enabling the sensors (there is an additional chip on that board that is used as a GPIO expander).

I'll change the comment to reflect that situation.